mlrun/runtimes/package/context_handler.py

```diff
@@ -0,0 +1,709 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import inspect
+import os
+import shutil
+from collections import OrderedDict
+from enum import Enum
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Type, Union
+
+import cloudpickle
+import numpy as np
+import pandas as pd
+
+from mlrun.datastore import DataItem
+from mlrun.errors import MLRunInvalidArgumentError, MLRunRuntimeError
+from mlrun.execution import MLClientCtx
+from mlrun.utils import logger
+
+
+# TODO: Move the `ArtifactType` to constants.py
```

Comment:
Is this `TODO` still relevant? If you already moved the code to a different place, it might be a good idea to follow the `TODO` here...

Reply:
Will be done as part of the packagers PR.